### PR TITLE
Signup frontend

### PR DESCRIPTION
**Pull request by emalamela**
### Summary

Implementation of the frontend of the signup screen. Validates the content and format of the input in the fields.

**Cards:** 
* Card: [09] https://trello.com/c/hguQNWUO

### Screenshots

device-2016-02-15-105922

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.wolox.emalamela.woloxnews">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.wolox.emalamela.woloxnews">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (juanignaciomolina) [inline app/src/main/AndroidManifest.xml]: You already have this permissions added

File: app/src/main/java/com/wolox/emalamela/woloxnews/views/HypertextView.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.wolox.emalamela.woloxnews.views;
+
+import android.content.Context;
+import android.text.method.LinkMovementMethod;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+public class HypertextView extends TextView {
```
Comment (juanignaciomolina) [inline app/src/main/java/com/wolox/emalamela/woloxnews/views/HypertextView.java]: Extracting this into its own `View` was an excellent idea! :smile:

File: app/src/main/res/layout/fragment_signup.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical">
+
+    <android.support.v7.widget.CardView
+        xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        card_view:cardBackgroundColor="?android:colorBackground"
```
Comment (juanignaciomolina) [inline app/src/main/res/layout/fragment_signup.xml]: Shouldn't this be an specific color provided by the guidelines?

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -26,4 +25,20 @@
...
+    <style name="wolox_news_terms_and_conditions">
+        <item name="android:gravity">center</item>
+    </style>
+
+    <style name="wolox_news_terms_and_conditions.text_black">
+        <item name="android:textColor">@color/wolox_news_black</item>
+    </style>
+
+    <!-- Signup -->
+
```
Comment (juanignaciomolina) [inline app/src/main/res/values/styles.xml]: Remove this empty line

File: app/src/main/java/com/wolox/emalamela/woloxnews/views/HypertextView.java (hunk 2 above)
Comment (emalamela) [inline app/src/main/java/com/wolox/emalamela/woloxnews/views/HypertextView.java]: giphy

File: app/src/main/res/layout/fragment_signup.xml (hunk 3 above)
Comment (emalamela) [inline app/src/main/res/layout/fragment_signup.xml]: Moved the background color to the colors resources file and set the default background for the app in the theme.

**Comment by juanignaciomolina** [comment]
@emalamela 
Squash both commits and gogogo QA :+1:

**Pull request merged**